RHEL6_7/packages/RemovedPackages/noauto_postupgrade.d/native_rpm_dep_fix.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+#RHSIGNED_PKGS="/var/cache/preupgrade/common/rpm_rhsigned.log"
+RHSIGNED_PKGS="rpm_rhsigned.log"
+kept_broken_rpms="kept_broken_rpms"
```

Comment:
I'm not sure why you need variable if the value is always just the variable name itself...